core/jobs/batch_jobs/skill_validation_jobs_test.py

```diff
@@ -0,0 +1,136 @@
+# coding: utf-8
+#
+# Copyright 2022 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.batch_jobs.skill_validation_jobs."""
+
+from __future__ import annotations
+
+from core.constants import constants
+from core.domain import skill_domain
+from core.jobs import job_test_utils
+from core.jobs.batch_jobs import skill_validation_jobs
+from core.jobs.types import job_run_result
+from core.platform import models
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import skill_models
+
+(skill_models, ) = models.Registry.import_models([models.NAMES.skill])
+
+
+class GetNumberOfSkillsWithInvalidRubricExplanationsJobTests(
+    job_test_utils.JobTestBase):
+
+    JOB_CLASS = (
+        skill_validation_jobs.GetNumberOfSkillsWithInvalidRubricExplanationsJob
+    )
+
+    def setUp(self):
+        super().setUp()
+
+        self.invalid_rubrics_1 = [
+            skill_domain.Rubric(
+                constants.SKILL_DIFFICULTIES[0],
+                ['<p> ' + 'a' * 350 + ' </p>']
```

Comment:
Can the explanation contain HTML?

Reply:
Yes I think. Because rubric explanations have html in `skill_domain_test`

Reply:
I think test files are not a good source of reliable information as they can contain mistakes, but it doesn't really matter here. Can you maybe check whether when we edit difficulties we use CKEditor?